frontend/src/components/Footer.jsx

```
@@ -0,0 +1,78 @@
+              <li>1234 Market Street</li>
+              <li>Suite 1000</li>
+              <li>San Francisco, CA 94103</li>
+              <li className="mt-4"><a href="mailto:contact@jobnode.com" className="hover:text-white transition-colors">contact@jobnode.com</a></li>
+            </ul>
+          </div>
+        </div>
+        
+        <div className="border-t border-gray-600 mt-8 pt-8 flex flex-col md:flex-row justify-between items-center">
+          <p className="text-gray-200 mb-4 md:mb-0">© {currentYear} JobNode. All rights reserved.</p>
+          <div className="flex space-x-6">
+            <a href="#" className="text-gray-200 hover:text-white transition-colors text-sm">Terms of Service</a>
```

Biome (1.9.4)

[error] Provide a valid value for the attribute href.

The href attribute should be a valid a URL
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)